Report/sections/2_design/semantics.tex

```diff
@@ -66,6 +66,14 @@ \subsection*{Type rules in \dazel{}}
 \ref{item:tiletype} is an important type: it is the type behind every tile in the generated games.
 \ref{item:arraytype} means that every type in the list will have a corresponding array type. This is to say that arrays are parametric - for example, an array could be an array of numbers.
 
+\subsection*{Dynamic and static binding}
+A variable must first be bound to a data type before it can be used in the program. 
```

Comment:
We don't explicitly declare types, which I think this sentence implies. When a variable is assigned, it receives the type that the expression evaluates to.

Reply:
This is the explanation of the general concept, not how it works in Dazel